packages/fluentui/react-docs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@fluentui/react-docs",
+  "description": "Tools for documenting FluentUI React components.",
+  "version": "0.44.0",
+  "author": "Levi Thomason <lethoma@microsoft.com>",
+  "bugs": "https://github.com/microsoft/fluent-ui-react/issues",
+  "bin": "dist/commonjs/index.js",
+  "dependencies": {
+    "@babel/core": "^7.7.7",
+    "@babel/parser": "^7.6.3",
+    "@babel/preset-typescript": "^7.7.7",
+    "@babel/runtime": "^7.7.6",
+    "@types/yargs": "^15.0.3",
+    "doctrine": "2.0.0",
+    "keyboard-key": "^1.0.1",
+    "lodash": "^4.17.15"
+  },
+  "devDependencies": {
+    "@fluentui/internal-tooling": "^0.43.0",
+    "@types/node": "^10.3.2",
+    "lerna-alias": "^3.0.3-0"
+  },
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://github.com/microsoft/fluent-ui-react/tree/master/packages/accessibility",
+  "jsnext:main": "dist/es/index.js",
```

Comment:
```suggestion
```

We should have there only CommonJS build I think

Reply:
Removed from package json